typescript/infra/src/govern/ProxiedRouterGovernor.ts

```diff
@@ -54,19 +56,29 @@ export class ProxiedRouterGovernor<
   }
 
   protected handleEnrolledRouterViolation(violation: RouterViolation) {
-    const remoteDomain = this.checker.multiProvider.getDomainId(
-      violation.remoteChain,
-    );
+    const domains: number[] = [];
+    const addresses: string[] = [];
+
+    for (const [remoteChain, expectedRouter] of Object.entries(
+      violation.routerDiff,
+    ) as [ChainName, Address][]) {
```

Comment:
nit: is the `as` needed? think the types should be fine

Reply:
resolved